google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStandardRunTask.java

```diff
@@ -46,5 +57,11 @@ public void execute(ProcessStartListener startListener) {
 
     CloudSdkAppEngineDevServer devServer = new CloudSdkAppEngineDevServer(sdkBuilder.build());
     devServer.run(runConfig);
+
+    UsageTrackerProvider.getInstance()
+        .trackEvent(GctTracking.APP_ENGINE_RUN)
+        .withLabel(runnerId != null ? runnerId : "")
```

Comment:
Minor thing, but you can replace this with a string helper from [Guava](http://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Strings.html#nullToEmpty(java.lang.String)): `Strings.nullToEmpty`. I believe apache commons provides something similar as well.